collectors/ciscoduo/collector.js

```diff
@@ -161,10 +162,15 @@ class CiscoduoCollector extends PawsCollector {
             };
         } else {
             //There is no next page concept for this API, So Setting up the next state mintime using the last log (Unix timestamp + 1).
+            // Added 5 min interval if there is no data and mintime is less than 5min. Wait for 5 min and then try again for next page.
+            const nowMoment = moment();
+            const nextMintime = moment(parseInt(nextPage * 1000));
+            const nextPollInterval = nowMoment.diff(nextMintime, 'seconds') > 300 ?
```

Comment:
I think we don't need it here because more data is available for collection. Let's keep using 1 sec here unless you've spotted any API throttling errors